backend/controllers/mainController.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * ====================================================================================================================
+ * This controller handles our routing functions. 
+ * We can use response.send() to send a message back to the client
+ * We can use request.params to see url paramaters (paramaters that are in the url, e.g. the "1" in api/entries/1)
+ * We can use request.body to see POST paramaters (paramaters that are sent in the POST request)
+ * ====================================================================================================================
+ */
+
+
+// First we load our dependencies. 
+var mongoose = require('mongoose'); //We need the mongoose library so we can work with our database.
+var Movie = require('../models/movie'); //We also need to load the Movie model.
+
+
+/*
+ * The table below lists the routes we will implement, just for reference.
+ * 
+ * Application Routes
+ * ------------------------------------------------------------------------------------------
+ * URI 						| HTTP ACTION 		| POST Body 		| Result
```

Comment:
You should use spaces instead of hard tabs. Otherwise it won't look like you expect.

![bad tabs](http://i.imgur.com/NQDp5uM.png)